### ISSUE TITLE
Animation seems broken

### ISSUE THREAD

**Issue by fedarko** [biocore/empress#294]
On trying to view an animation, no toast shows up at the end, and I get the following error in the console:

```
biom-table.js:401 Uncaught Error: No samples have "undefined" as their value in the "days-since-experiment-start" gradient sample metadata column.
    at BIOMTable.getGradientStep (biom-table.js:401)
    at Empress.getGradientStep (empress.js:1886)
    at Animator.retriveFrame (animator.js:321)
    at Animator._collectFrame (animator.js:209)
    at loop (animator.js:256)
```

Not sure what is causing this right now

**Comment by kwcantrell** [comment]
Are you on the master branch? I cannot recreate this error. If you are working off of your bar plot PR, then I might know the cause. Would it be possible to share the .qza you used to create the .qzv?

**Comment by fedarko** [comment]
This was with the barplot PR branch (merged in with the master changes), but I just pulled the latest changes from master and switched into a branch without the barplot changes and it looks like this is still present.

Here's the QZV (I had to rename it to `.zip` in order to get GitHub to accept it)

[empress-tree.zip](https://github.com/biocore/empress/files/5024736/empress-tree.zip)

**Comment by kwcantrell** [comment]
You are right, this was an error I introduced with clade collapse. (I slightly changed the way the animator keeps track of frames). I didn't merge the clade collapse yet which is why I did not get the error. I am now pushing a fix.

**Comment by fedarko** [comment]
Closed by #296.
